odata-url-conventions/5 Query Options.md

```diff
@@ -1676,6 +1676,17 @@ $filter=[FirstName,LastName] in [["John","Doe"],["Jane","Smith"]]
 ```
 :::
 
+Transient entities are represented as complex literals with their entity set
```

Comment:
Entity set / context URL is optional. Could also specify the type in case of inheritance.

Reply:
Can also use structured literal that represents an entity reference.

Reply:
For non-transient, could presumably do any of the following:
1.    {"@id":"Products(1)}                                   // this is actually how I would have expected it to be represented
2.    {"@context":"#Products","ProductId":1}    // alternate representation specifying entity set and key
3.    $root/Products(1)                                      // path syntax